editor/src/components/canvas/dom-lookup.ts

```diff
@@ -165,32 +186,103 @@ export function findFirstParentWithValidElementPathInner(
         target.parentElement,
         allowDescendantsFromPath,
         parentSceneValidPathsCache,
+        metadata,
+        point,
       )
     }
   }
 }
 
+function extendValidPathsWithNonDomParent(
+  domPath: {
+    static: string
+    staticPath: ElementPath
+    dynamic: ElementPath
+  },
+  nonDomPath: ElementPath,
+): ElementPath | null {
+  // check if there is a dom path which is a descendant of a non-dom path: then the
+  // non-dom path should be a valid target too
```

Comment:
This comment says that the "...non-dom path should be a valid target too", but it doesn't return `nonDomPath`, which seems odd.